app/controllers/api/base_controller.rb

```diff
@@ -60,6 +60,12 @@ def current_version
       params[:api_version]
     end
 
+    def update_last_used_at!
```

Comment:
I suggest to use another name for the callback for semantic purposes like `update_token_last_used` as it currently looks like a delegated method on the `auth_token`.